cypress/integration/user-flows/scenario_001.spec.js

```diff
@@ -10,7 +10,6 @@ describe("I am a new SM user (with Next) who wants to create a Custom Type with
     cy.clearLocalStorageSnapshot();
     cy.cleanSliceMachineUserContext();
     cy.task("clearDir", path.join(appPath, "customtypes"));
-    cy.task("clearDir", path.join(appPath, "slices"));
```

Comment:
I think this was there so tests can be rerun

Reply:
riiight, wrong file :/